addons/im_livechat/static/src/core/common/thread_actions.js

```diff
@@ -0,0 +1,28 @@
+import { threadActionsRegistry } from "@mail/core/common/thread_actions";
+
+const actinConfig = threadActionsRegistry.get("view-contact");
+
+threadActionsRegistry.add("livechat-visitorPartner-contact", {
+    ...actinConfig,
+    condition(component) {
+        return (
+            component.thread?.channel_type === "livechat" &&
+            component.thread?.livechatVisitorMember?.persona.type === "partner"
+        );
+    },
+    open(component) {
+        const visitor = component.thread.livechatVisitorMember;
+        component.actionService?.doAction(
```

Comment:
The button should probably not be shown if there is no `actionService`.

Is this in the wrong bundle? Maybe code was meant to be in `web` folder?